### PR TITLE
Return statment bugfix

### PR DESCRIPTION
**Pull request by jundl77**
Added the missing return statement type and also added chained functions and functions being called inside parameters

### CODE REVIEW

File: src/main/java/xyz/exemplator/exemplator/parser/java/JavaParser.java [hunk 1]
```diff
@@ -82,6 +82,12 @@ public JavaParser(CompilationUnit cu) {
                 .findAny()
                 .isPresent();
 
+        if (cu.toString().contains("package com.donnfelker.android.bootstrap.core;\n" +
```
Comment (LeanderK) [inline src/main/java/xyz/exemplator/exemplator/parser/java/JavaParser.java:85]: why is this merged?

File: src/main/java/xyz/exemplator/exemplator/parser/java/UsageFinder.java [hunk 2]
```diff
@@ -176,18 +125,84 @@ public UsageFinder(CompilationUnit cu, TypeDeclaration type, Command command, bo
...
             }
         }
         return usages;
     }
 
+    private void handleMethodExpressionStmt(Set<LeftQualifier> fieldQualifiers, boolean identifierOnlyEnabled, Set<LeftQualifier> inheritedLocalVars, Set<LeftQualifier> localVars, Set<LeftQualifier> localVarsBlocking, List<Selection> usages, ExpressionStmt statement) {
+        handleMethodExpression(fieldQualifiers, identifierOnlyEnabled, inheritedLocalVars, localVars, localVarsBlocking, usages, statement.getExpression());
+    }
+
+    private void handleMethodExpression(Set<LeftQualifier> fieldQualifiers, boolean identifierOnlyEnabled, Set<LeftQualifier> inheritedLocalVars, Set<LeftQualifier> localVars, Set<LeftQualifier> localVarsBlocking, List<Selection> usages, Expression expression) {
```
Comment (LeanderK) [inline src/main/java/xyz/exemplator/exemplator/parser/java/UsageFinder.java:142]: i don't like the method, it should not return void but a Set of Selections

**Review by LeanderK — COMMENTED** [review]
not so fast...

File: src/main/java/xyz/exemplator/exemplator/parser/java/JavaParser.java (hunk 1 above)
Comment (LeanderK) [reply src/main/java/xyz/exemplator/exemplator/parser/java/JavaParser.java:85]: ah, you deleted it in the master

File: src/main/java/xyz/exemplator/exemplator/parser/java/UsageFinder.java (hunk 2 above)
Comment (jundl77) [reply src/main/java/xyz/exemplator/exemplator/parser/java/UsageFinder.java:142]: you can change it

Comment (LeanderK) [reply src/main/java/xyz/exemplator/exemplator/parser/java/UsageFinder.java:142]: this is your job if you introduced the method

**Pull request merged**